.github/workflows/ci.yml

```diff
@@ -181,7 +181,7 @@ jobs:
           terraform-plan-file: plan
 
       - name: Analyze Terraform Plan
-        uses: byu-oit/github-action-tf-plan-analyzer@v2
+        uses: byu-oit/github-action-tf-plan-analyzer@v2.0.1
```

Comment:
👀 I'd like to think we're effectively treating `v2` as `^2.0.0`, so this changes us from `^2.0.0` to `2.0.1` (without the `^`).

Reply:
So per [this conversation in Slack](https://byu-oit.slack.com/archives/CQ2BE663T/p1605907223154800), we should change this back to `v2`, and update `github-action-tf-plan-analyzer` to use `v2` as a branch that gets updated with minor/patch changes.